spec/requests/api/single_page_spec.rb

```diff
@@ -110,6 +110,40 @@
       expect(response).to have_http_status(:ok)
     end
 
+    it 'returns the aggregated values for each metric' do
+      get "/single_page/#{base_path}", params: { from: '2018-01-01', to: '2018-01-31' }
+
+      body = JSON.parse(response.body)
+
+      expected = {
+        'time_series_metrics' => a_collection_including(
+          a_hash_including(
+            'name' => 'upviews',
+            'total' => 30
+            )
+          )
+      }
+      expect(body).to include(expected)
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'returns the aggregated recalculated value for Satisfaction Score' do
```

Comment:
`aggregated recalculated value` is confusing -  maybe simplify the test title to `returns the satisfaction score for time period`